### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by zen0bit**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

## Trying again

pull request #33537

### CODE REVIEW

**Comment by Duncaen** [comment]
1. You don't have to reopen PRs, force push to the branch instead of spamming new PRs.
2. Switching maintainer and upstream to your own without any reason/input/comment seems weird.

File: common/travis/fetch_upstream.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@ elif command -v git >/dev/null 2>&1; then
 fi
 
 /bin/echo -e '\x1b[32mFetching upstream...\x1b[0m'
-$GIT_CMD fetch --depth 200 git://github.com/void-linux/void-packages.git master
+$GIT_CMD fetch --depth 200 git://github.com/oSoWoSo/void-packages.git master
```
Comment (paper42) [inline common/travis/fetch_upstream.sh:12]: That's not how this works, rebase on top of our master, remove these CI changes and make sure to have only one commit with proper description.

Comment (zen0bit) [reply common/travis/fetch_upstream.sh]: reverted

**Comment by paper42** [comment]
I looked at your fork and it is only one commit ahead of the archived upstream - https://github.com/oSoWoSo/clearine/commit/a59184e0e7c9ed49437c255e37944457bf531690. The commit description is useless, is this fixing a bug? Could you describe it? If it is, I think we should just include patch in this repository and not change the upstream.

cc @steinex - the package maintainer

**Comment by zen0bit** [comment]
I am already trying...
First try here
https://github.com/oSoWoSo/void-packages/tree/clearine-patch

Not tested yet

**Comment by paper42** [comment]
Please test the changes on your computer first, verify that git log shows what you expect and then force push here.

> The commit description is useless, is this fixing a bug? Could you describe it?

Could you describe what are you trying to achieve?

**Comment by zen0bit** [comment]
I cannot build it anymore even from official template...

**Comment by paper42** [comment]
```
>>> a = "123"
>>> a is "123"
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
True
>>> a == "123"
True
```
So this PR is not fixing anything other than syntax warnings, if I am wrong, please leave a comment. I asked multiple times what this PR is supposed to fix, but you didn't respond.

**Comment by zen0bit** [comment]
Yes

**Comment by paper42** [comment]
> ```
> >>> a = "123"
> >>> a is "123"
> <stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
> True
> >>> a == "123"
> True
> ```

This actually works only in some cases, if you describe what is actually happening, what is the bug and how can we reproduce this, we can reopen this PR.

**Pull request closed without merging**